core/audits/bf-cache.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {Audit} from './audit.js';
+import * as i18n from '../lib/i18n/i18n.js';
+import {NotRestoredReasonDescription} from '../lib/bfcache-strings.js';
+
+/* eslint-disable max-len */
+const UIStrings = {
+  /** TODO */
+  title: 'Back/forward cache is used',
+  /** TODO */
+  failureTitle: 'Back/forward cache is not used',
+  /** TODO */
+  description: 'The back/forward cache can speed up the page load after navigating away.',
```

Comment:
Let's emphasize how big a deal this can be.

web.dev says:

> Chrome usage data shows that 1 in 10 navigations on desktop and 1 in 5 on mobile are either back or forward. With bfcache enabled, browsers could eliminate the data transfer and time spent loading for billions of web pages every single day!